rust-semantics/rust-common-syntax.md

```diff
@@ -257,8 +257,7 @@ https://doc.rust-lang.org/reference/items/extern-crates.html
                               | ByteLiteral | ByteStringLiteral | RawByteStringLiteral
                               | CStringLiteral | RawCStringLiteral
                               | IntegerLiteral | FloatLiteral
-                              | "true" | "false"
-
+                              
```

Comment:
Why is there a diff for this line? It should be identical with the empty line in the original file.

Reply:
It was meaningless spaces. I've just removed it.